src/recording/macromanager.h

```diff
@@ -74,9 +84,19 @@ class MacroManager : public RecordingManagerBase {
     void stopRecording() override;
     bool isRecordingActive() override;
 
-    void appendHotcueJump(int channel, double origin, double target);
+    /// Tries to append this cue jump to the currently recorded Macro.
+    /// Returns true if the currently recorded Macro was changed - so only if
+    /// recording is active and the channel handle matches.
```

Comment:
Why does this information need to be passed back to EngineBuffer? Can this return `void`?

Reply:
Yes it can return void, I thought that might be useful in the future for controller feedback or something, and it wasn't a big deal to implement.

Reply:
Don't implement code that you don't have a plan to use. When someone else looks at this in 5 years they may wonder the same question I did but not be sure if it's okay to change the return type.